public/video-ui/src/components/Presence.js

```diff
@@ -1,7 +1,10 @@
 import React from 'react';
 
 export class Presence extends React.Component {
-  state = null;
+  state = {
+    client: null,
+    visitors: []
```

Comment:
this felt like the simplest thing to do, rather than throw it to the store... because I'm lazy 😄

Reply:
It's easy enough to push up if it's needed elsewhere. Local state is for this kind of thing anyway